staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -384,13 +384,11 @@ type KubeletConfiguration struct {
 	// +optional
 	NodeStatusReportFrequency metav1.Duration `json:"nodeStatusReportFrequency,omitempty"`
 	// nodeLeaseDurationSeconds is the duration the Kubelet will set on its corresponding Lease,
-	// when the NodeLease feature is enabled. This feature provides an indicator of node
-	// health by having the Kubelet create and periodically renew a lease, named after the node,
-	// in the kube-node-lease namespace. If the lease expires, the node can be considered unhealthy.
+	// NodeLease provides an indicator of node health by having the Kubelet create and periodically renew a lease, named
+	// after the node, in the kube-node-lease namespace. If the lease expires, the node can be considered unhealthy.
```

Comment:
While you're fixing that comma, can you reflow this to be the same line length as the surrounding comments?